students/PeterB/Session3/slicing.py

```diff
@@ -0,0 +1,36 @@
+# slicing
+# first and last
+seq = '3, 2, 1'
+def exchange_first_last(seq):
+    first = seq[-1]
+    mid = seq[1:-1]
+    last = seq[0]
+    return print([first] + mid + [last])
+
+# with every other item removed
+
+my_list = "1, 2, 3, 4, 5, 6"
+def remove_every_other(my_list):
+    return my_list[::2]
+
+# first four last four
+
+def first_last_four(sequence):
+    first_four = sequence[:4]
+    last_four = sequence[-4:]
+    new_sequence = first_four + last_four
```

Comment:
The goal of this is: with the first 4 and the last 4 items removed, and then every other item in the remaining sequence.

You want to remove the ones you have shown here and then every other item in the middle.